### ISSUE TITLE
Umowa własna Inpost - wysyłam z Allegro

### ISSUE THREAD

**Issue by caroteczek** [allegro/allegro-api#10005]
Dzień dobry, 

czy coś się zmieniło po stronie api Allegro. Nie mogę nadać listu ze względu na bład:
![image](https://github.com/user-attachments/assets/011ae7ea-2b0a-415f-ad5e-d60afe8882f4)
Mam dodaną umowę własną do konta Allegro, i połączona jestem z kontem Allegro po przez rejestrację aplikacji, a mimo to integracja w której pracuję nie widzi umów własnych:
Czy to nie jest związane z tą zmianą:
![image](https://github.com/user-attachments/assets/43ca0051-73c6-4f2b-9c35-e41869cb1d26)

**Comment by AureliuszBrussy** [comment]
Tak, 26 września wprowadziliśmy zmiany w nadawaniu przesyłek na umowach własnych. Więcej możesz przeczytać [w komunikacie](https://developer.allegro.pl/news/wysylam-z-allegro-26-wrzesnia-wprowadzimy-zmiany-w-nadawaniu-przesylek-na-umowach-wlasnych-nnxEA20gdud).

**Comment by kepesian** [comment]
Nadaję na umowie własnej Inpost, w danych zamówienia id metody dostawy to ""2488f7b7-5d1c-4d65-b85c-4cbcf253fd93" co jest zgodne z nazwą usługi "Allegro Paczkomaty InPost".
Generuję etykiete i realizuję przesyłkę poprzez własną apkę i API Shipx, a następnie chcę wpisać numer przesyłki  do zamówienia poprzez API Allegro ( enpoint) "/order/checkout-forms/{$orderID}/shipments";
Pytanie gdzie pobrać, odnaleźć prawidłowy dla tego rozwiązania CarrierId , gdy nie ma go w zasobie "/shipment-management/delivery-services" bo nie ma umowy własnej. Odnalazłem w GUI SC w dodawaniu numeru przesyłki, że jest to "INPOST" ale gdzie to odnaleźć lub przez jaki zasób są dostępne CarrierId metod wysyłki nie należących do usera, tak jak w tym wypadku.

**Comment by PrzemyslawLukanowski** [comment]
@kepesian Dostępne wartości dla carrierId sprawdzisz za pomocą [GET /order/carriers](https://developer.allegro.pl/documentation#operation/getOrdersCarriersUsingGET).

**Comment by kepesian** [comment]
OK, ale jak na podstawie zamówienia i id metody zawartej w danych zamówienia odnaleźć właściwy carrierId gdy nie ma go w umowach własnych. Innymi slowy , aktualizacja zamówienia o wpisaniu numeru przesyłki wymaga podania carrierId. Zamówienie jest z metodą 2488f7b7-5d1c-4d65-b85c-4cbcf253fd93, która nazywana jest "Allegro Paczkomaty InPost". W jaki sposób odnaleźć właściwy carrierId na podstawie tych danych dostępnych w zamówieniu, tak aby podać właściwy carrierId przy wywołaniu zasobu POST "/order/checkout-forms/{$orderID}/shipments"; ?

**Comment by PrzemyslawLukanowski** [comment]
Nie mamy w tym przypadku udostępnionej metody, która pozwoli Ci jednoznacznie określić carrierId na podstawie metody dostawy. W takim przypadku możesz po swojej stronie oprzeć się na nazwie metody, występującej w niej nazwie przewoźnika, i tym sposobem dopasować odpowiednią wartość z GET /order/carriers.
